.github/workflows/test_all.yml

```diff
@@ -1,35 +1,33 @@
 name: Run all Qadence tests on schedule.
-
-on:
-  schedule:
-    # 03:00 every Saturday morning
-    - cron: '0 3 * * 6'
-  workflow_dispatch: {}
+on: push
 
 concurrency:
   group: all-${{ github.head_ref || github.run_id }}
   cancel-in-progress: true
 
+
+
 jobs:
-  test_qadence_ubuntu:
-    name: Test Qadence (ubuntu)
-    runs-on: ubuntu-latest
+  test_qadence_windows_mac:
+    name: Test Qadence (Windows, MacOS)
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.11"]
+        os: [macos-latest, windows-latest]
```

Comment:
lets re-add ubuntu-latest and merge?

Reply:
yeah. Notice I'm running the full tests on ubuntu but only the fast tests on mac and windows. But I think that should be ok